frontend/src/components/program/picasso/Picasso.vue

```diff
@@ -374,14 +357,6 @@ export default {
       const color = scheduleEntry.activity().category().color
       return isCssColor(color) ? color : color + ' elevation-4 v-event--temporary'
     },
-    isActivityLoading(scheduleEntry) {
-      return (
-        !scheduleEntry.tmpEvent &&
-        (this.activitiesLoading ||
-          this.categoriesLoading ||
-          scheduleEntry.activity()._meta.loading)
-      )
-    },
```

Comment:
Is this the same as `isCategoryLoading`? Should it keep its name?

Reply:
Not sure what you mean. I deleted this method and moved it to the PicassoEntry component, where it was further simplified. Not sure what you want me to rename..?

Reply:
You can see the `isCategoryLoading` in `Picasso.vue` just below the deleted `isActivityLoading`. To me it looks the same.

Reply:
No that is okay. I just wondered about the naming of the method.

I think, now we could even render the schedule entry if the category or the activity has not been loaded yet. So for slow connections we could update the picasso entry each time we get some more information like activity title and then category informations

Reply:
Actually I think we can also easily move the colors to PicassoEntry. Give me another minute

Reply:
Done. All methods which calculated schedule entry specific data are moved to PicassoEntry.vue

Reply:
If we have no "dependencies" left from picasso, I think it would make sense to display number, title and category incrementally in activityname. I would love to see all entries, before everything is loaded. Often I already know which activity I'm going to click.